cosmos/cache.py

```diff
@@ -400,3 +412,72 @@ def create_cache_profile(version: str, profile_content: str) -> Path:
     profile_yml_path = profile_yml_dir / DEFAULT_PROFILES_FILE_NAME
     profile_yml_path.write_text(profile_content)
     return profile_yml_path
+
+
+def is_cache_package_lockfile_enabled(project_dir: Path) -> bool:
```

Comment:
Slightly confused with the name of this method. It appears that the implementation in this method checks whether the project has the PACKAGE_LOCKFILE_YML file, but the name of the method implies that we are checking if caching is enabled or not.

Reply:
We could follow the same pattern we're using for enabling/disabling other cache mechanisms:
https://github.com/astronomer/astronomer-cosmos/blob/60148ceb311e41c5c27a7be1b584bc5fcaafbcfe/cosmos/settings.py#L15-L17

And it would justify keeping this name

Reply:
Added a config to enable and disable it with default value True to enable